### DISCUSSION TITLE
SMART on FHIR app launch

### DISCUSSION THREAD

**Discussion by dattachandan** [medplum/medplum#1222] [Ideas]
Hi team

Greetings from Middle Earth, NZ!

I was keen to know if there is any sample of the SMART on FHIR launch of the Medplum app on the Medplum CDR?

- https://github.com/medplum/medplum/blob/main/packages/examples/README.md
- https://github.com/medplum/medplum/blob/main/packages/mock/README.md

**Comment by reshmakh** [comment]
We do not yet have an example app in the repo, but we do have #1230 documenting the open issues and will create the documentation and then the sample.

**Comment by reshmakh** [comment]
@dattachandan - we have just released a guide for [SMART app Launch](https://www.medplum.com/docs/integration/smart-app-launch) and have completed the Inferno live testing for it.  How does this look to you? Let us know here or in the [Discord](https://discord.gg/UBAWwvrVeN)
